src/cljConnection.ts

```diff
@@ -136,12 +136,10 @@ const startNRepl = (): void => {
         .then(connectionInfo => nreplConnection = connectionInfo)
         .then(() => nreplClient.test(nreplConnection))
         .then(stopLoadingAnimation)
-        .then(() => saveConnection(nreplConnection))
```

Comment:
This change causes the rejection listener to fire correctly when the nREPL process terminates before a port can be found.